docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -679,6 +690,14 @@
 			<p>Comming soon</p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m57visionapplicationsample.png" width="150" /></td>
+			<td>
+			<p><strong>VisionApplicationSample</strong></p>
+			<p>VisionApplicationSamples shows how to use implement Barcode Generation/Detection, Face Detection, and Image Recognition.</p>
```

Comment:
Can we change to (remove VisionApplicationSamples from the line as it is confusing to read because of the name):

This sample application shows how to implement: Barcode Generation/Detection, Face Detection, and Image Recognition.